File: PasuBleSample/ButterKnifeTesting/src/main/java/pl/emget/butterknifetesting/MainActivity.java
package pl.emget.butterknifetesting;

import android.app.Activity;
import android.os.Bundle;
import android.widget.Button;
import android.widget.TextView;

import butterknife.BindView;
import butterknife.ButterKnife;

public class MainActivity extends Activity {

    @BindView(R.id.text1)
    TextView textView1;
    @BindView(R.id.button1)
    Button button1;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ButterKnife.bind(this);

        textView1.setText("KKKK");
    }
}

File: NewASApp/app/src/main/java/com/example/psuszek/myapplication/MainActivity.kt
package com.example.psuszek.myapplication

import android.app.Activity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView
import butterknife.BindView
import butterknife.ButterKnife


class MainActivity : Activity() {

    //val title: TextView by bindView(R.id.text1)

    var aa: Button?=null

    //@BindView(R.id.text1) public var title: TextView? = null

    @BindView(R.id.text1)
    lateinit var title2 : TextView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


        savedInstanceState.toString()

        ButterKnife.bind(this)

        //title = findViewById(R.id.seekBar1)

        title2.setText("kkk");

//        title2.setOnClickListener(new OnClickListener(){
//
//        })



    }
}
